### PR TITLE
feat(structure): create new routes 'home' and 'show-forms'

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

File: caluma-demo/app/components/navbar.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Component from "@ember/component";
+
+export default class NavbarComponent extends Component {}
```
Comment (anehx) [inline caluma-demo/app/components/navbar.js]: If you add `"template-only-glimmer-components": true` to `config/optional-features.js` you can delete this file :smile:

File: caluma-demo/app/routes/show-forms.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import Route from "@ember/routing/route";
+import { queryManager } from "ember-apollo-client";
+import gql from "graphql-tag";
+import { decodeId } from "ember-caluma/helpers/decode-id";
+
+export default Route.extend({
```
Comment (anehx) [inline caluma-demo/app/routes/show-forms.js]: New syntax please :heart:

File: caluma-demo/app/templates/show-forms.hbs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{{#if model}}
+    {{cf-content documentId=model}}
+{{/if}}
```
Comment (anehx) [inline caluma-demo/app/templates/show-forms.hbs:3]: Wrong indentation: check with `yarn lint:hbs`!

**Review by anehx — CHANGES_REQUESTED** [review]

File: caluma-demo/app/components/navbar.js (hunk 1 above)
Comment (JohnL17) [reply caluma-demo/app/components/navbar.js]: Nice, thanks!

File: caluma-demo/app/routes/show-forms.js (hunk 2 above)
Comment (JohnL17) [reply caluma-demo/app/routes/show-forms.js]: Done.

File: caluma-demo/app/templates/show-forms.hbs (hunk 3 above)
Comment (JohnL17) [reply caluma-demo/app/templates/show-forms.hbs:3]: Done.

File: caluma-demo/app/templates/components/navbar.hbs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<nav class="uk-navbar-container" uk-navbar>
+  <div class="uk-navbar-left">
+    <ul class="uk-navbar-nav">
+      <li><LinkTo @route="home">{{t "Home" }}</LinkTo></li>
+      <li><LinkTo @route="form-builder">{{t "Form-Builder" }}</LinkTo></li>
+      <li><LinkTo @route="show-forms">{{t "Show Forms" }}</LinkTo></li>
```
Comment (anehx) [inline caluma-demo/app/templates/components/navbar.hbs]: Those translations won't work, you have to use keys like `nav.home` or `nav.form-builder` and then add those translations in `translations/en.yml`

File: caluma-demo/app/templates/home.hbs [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<div class="uk-container-xl-large uk-text-center">
+  <h2>{{t "Welcome to the Caluma-Demo" }}</h2>
+  <p>
+    {{t "On this page you can try Caluma. You can create forms, fill them out and view them directly." }}
+  </p>
```
Comment (anehx) [inline caluma-demo/app/templates/home.hbs:5]: Same thing here..

**Review by anehx — CHANGES_REQUESTED** [review]

File: caluma-demo/app/templates/components/navbar.hbs (hunk 4 above)
Comment (JohnL17) [reply caluma-demo/app/templates/components/navbar.hbs]: I did this already but with the wrong syntax. I have corrected it.

File: caluma-demo/app/templates/home.hbs (hunk 5 above)
Comment (JohnL17) [reply caluma-demo/app/templates/home.hbs:5]: Done.

**Review by anehx — APPROVED** [review]

**Pull request merged**